src/common/security/robot/context.go

```diff
@@ -32,16 +35,18 @@ type SecurityContext struct {
 	isSystemLevel bool
 	ctl           project.Controller
 	policy        []*types.Policy
-	evaluator     evaluator.Evaluator
+	proEvaluator  evaluator.Evaluator
+	sysEvaluator  evaluator.Evaluator
```

Comment:
No need these two evaluators here. evaluator.Evaluator is an interface, there is an evaluator.Evaluators which can include multi evaluators.

Reply:
OK, do not know there is an evaluator.Evaluators, will use it.